packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -18,6 +19,16 @@ const propTypes = {
   * Array of links to show for the content of the help button
   */
   helpItems: PropTypes.array,
+  alerts: PropTypes.shape({
+    alertList: PropTypes.arrayOf(
+      PropTypes.shape({
+        messageID: PropTypes.string,
+        messageText: PropTypes.oneOfType([
```

Comment:
nit: Maybe change these props to alertId and alertMessage since more than just text is supported.